packages/Test.candy.hints

```diff
@@ -0,0 +1,5 @@
+Hint { kind: Value, text: " # [ 0: 110, 10: 97, 16: 50, 20: 111, 24: 44, 4: 44, 6: 103, 3: 101, 7: 101, 13: 101, 22: 97, 21: 110, 9: 77, 23: 115, 17: 50, 14: 108, 19: 74, 5: 97, 8: 10, 11: 114, 25: 50, 12: 99, 26: 50, 18: 10, 1: 97, 15: 44, 2: 109 ]", position: Position { line: 0, character: 28 } }
```

Comment:
I think we can `.gitignore` the generated `*.candy.*` files

Reply:
Oh, definitely, good idea. I have several of these files which I always consciously ignore in GitHub Desktop, but seems like one slipped through.